### PR TITLE
feature/add-github-url

### PR DESCRIPTION
**Pull request by isometimescode**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [x] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.

All related to our findings in #17:
* Adding a GitHub URL to the template so we can use a different GitHub repo name and CDP instance name. [Related comment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment/issues/17#issuecomment-798779840)
* Fix bug in build [Related comment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment/issues/17#issuecomment-798779407)

- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Comment by isometimescode** [comment]
@JacksonMaxfield After using the cookiecutter to make this [test repo](https://github.com/isometimescode/cdp-testing) I no longer have broken images for the build process. However, I'm not sure if this change is what you were imagining when we were talking about adding a GitHub URL earlier, so lmk if you had a different solution in mind.

File: cookiecutter.json [hunk 1]
```diff
@@ -3,5 +3,7 @@
     "municipality_slug": "{{ cookiecutter.municipality.lower().replace(' ', '-').replace('_', '-') }}",
     "python_municipality_slug": "{{ cookiecutter.municipality_slug.replace('-', '_') }}",
     "maintainer_full_name": "Jackson Maxfield Brown",
-    "hosting_github_username_or_org": "CouncilDataProject"
+    "hosting_github_url": "https://github.com/CouncilDataProject/example",
+    "hosting_github_username_or_org": "{{ cookiecutter.hosting_github_url.replace('https://github.com/', '').split('/')[0] }}",
+    "hosting_github_reponame": "{{ cookiecutter.hosting_github_url.replace('https://github.com/', '').split('/')[1] }}"
```
Comment (JacksonMaxfield) [inline cookiecutter.json:8]: I think I would do this in the opposite order:

1. `hosting_github_username_or_organization`
2. `repository_name`
3. string combine to have `hosting_github_url`

File: {{ cookiecutter.hosting_github_reponame }}/.github/workflows/build-main.yml [hunk 2]
```diff
@@ -28,9 +28,9 @@ jobs:
     - name: Lint and Format Python
       run: |
         cd python/
-        flake8 cdp_example_backend --count --verbose --show-source --statistics
-        black --check cdp_example_backend
-  
+        flake8 cdp_{{ cookiecutter.python_municipality_slug }}_backend --count --verbose --show-source --statistics
+        black --check cdp_{{ cookiecutter.python_municipality_slug }}_backend
+
```
Comment (JacksonMaxfield) [inline {{ cookiecutter.hosting_github_reponame }}/.github/workflows/build-main.yml:33]: THANK YOU!

**Review by JacksonMaxfield — COMMENTED** [review]
!! Thanks @isometimescode! This is a great update.

I have one small change request (or discussion if you can convince me otherwise) of just flipping the order you ask for details.

File: cookiecutter.json (hunk 1 above)
Comment (isometimescode) [reply cookiecutter.json:8]: I went back and forth on how best to ask for this info. I like your suggestion @JacksonMaxfield ! I'll push an update later today.

**Review by JacksonMaxfield — APPROVED** [review]
Thanks again!

**Pull request merged**